### ISSUE TITLE
support blob image type

### ISSUE THREAD

**Issue by gmareater** [sanjayV/ng-image-slider#26]
hi
thanx for you nice work
When i pass base64, it says "provided images array not valid" 
here is the code, and i tried 3 different combination to pass url to image array
```
                this.sub = this.service.DownloadFile(file).subscribe(blob => {
                    const reader = new FileReader();

                    // This fires after the blob has been read/loaded.
                    reader.addEventListener('loadend', (e) => {
                        //console.log("fileReader loaded", e);
                        var result = (<any>e.srcElement).result;

                        //let url = this.sanitizer.bypassSecurityTrustUrl('data:image/' + this.File.FileType + ';base64,' + result);
                        let url = this.sanitizer.bypassSecurityTrustUrl(result);

                        this.Images.push({
                            image: result,
                            alt: file.FileTitle,
                            title: file.FileTitle
                        });

                    });
                    reader.readAsDataURL(blob);
                });

```

**Comment by sanjayV** [comment]
HI @gmareater 

can you share sample base64 string here in which you facing issue..

**Comment by sanjayV** [comment]
are you including **data:image/jpeg;base64,**  text with your base64 string?

**Comment by gmareater** [comment]
thanx for you answer, i forked your git and found out what was wrong,
you took whole array at first and process it, but i get my images after form load (they should download first), so i have and empty array that passed to your components, but  when I fill array, there is no process on new ones, 
I changed my code to wait for array to fully loaded then pass it, and it worked nicely.
but currently i have a similar issue, if my end user add new image to collection they wont displayed, so currently i'm working on it :)

**Comment by sanjayV** [comment]
@gmareater I will also try to implement it in next version :)

**Comment by sanjayV** [comment]
@gmareater 
fixed this issue in version [2.3.0](https://www.npmjs.com/package/ng-image-slider)
